lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -716,12 +786,7 @@ class TraceProcessor {
     // LCP comes from the latest `largestContentfulPaint::Candidate`, but it can be invalidated
```

Comment:
While you're here, could you move the details of this comment to `computeValidLCPAllFrames`, and add a comment here like "this function accepts events spanning multiple frames, but this usage will only provide events from this frame"